requirements-dev.txt

```diff
@@ -1,3 +1,3 @@
-pylint==2.6.0
-responses==0.12.0
-unittest-xml-reporting==3.0.4
+pylint>=2.6.0
+responses>=0.12.0
+unittest-xml-reporting>=3.0.4
```

Comment:
Oof, I hate seeing stuff like this. Why do we have to do it? We can't just pin to a known version?

Reply:
Seems fine to pin to specific dev deps, as I'm guessing that will not interfere with Jake being used to scan projects. I will push some changes to lock these dev dep version down (as they used to be).

Reply:
Dev deps locked: 5189624